synapse/events/snapshot.py

```diff
@@ -112,6 +112,7 @@ class EventContext:
 
     _current_state_ids = attr.ib(default=None, type=Optional[StateMap[str]])
     _prev_state_ids = attr.ib(default=None, type=Optional[StateMap[str]])
+    _outlier = attr.ib(default=False, type=bool)
```

Comment:
please add documentation for any new fields to the docstring, though I'm a bit surprised this is needed.